src/components/errors/styles/FatalErrorBoundary.module.css

```diff
@@ -1,61 +1,34 @@
-.mask {
-    position: absolute;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    overflow: auto;
-    overflow-y: auto;
-    color: #000000;
-    background-color: #f4f6f8;
-    display: flex;
-    min-width: 640px;
-    min-height: 480px;
-    align-items: center;
-    justify-content: center;
-}
-
-.container {
-    text-align: center;
-    color: black;
-}
-
-.icon {
-    width: 96px;
-    height: 96px;
-    color: iconColor;
-    margin-bottom: var(--spacers-dp24);
-}
-
-.message {
-    font-size: 24px;
-    margin-bottom: var(--spacers-dp24);
+.notice {
+    font-family: Roboto, sans-serif;
+    color: rgb(33, 41, 52);
+    line-height: 20px;
+    width: 500px;
 }
 
 .link {
-    font-size: 18px;
     text-decoration: underline;
     cursor: pointer;
-    margin-bottom: var(--spacers-dp24);
```

Comment:
why not use the spacers?

Reply:
Spacers can't be used as they are defined inside FatalErrorBoundary: https://github.com/dhis2/maps-app/blob/master/src/components/app/App.js#L47

I would like to keep the error boundary there as there might also be fatal error in the CSS setup.